corehq/apps/locations/static/locations/ko/location_drilldown.async.js

```diff
@@ -107,6 +107,11 @@ function LocationModel(data, root, depth, func, withAllOption) {
   this.func = typeof func !== 'undefined' ? func : LocationModel;
   this.withAllOption = typeof withAllOption !== 'undefined' ? withAllOption : true;
 
+  this.enable = function() {
```

Comment:
"enable" is a verb, which makes `something.enable()` sound like it's enabling something. However, the body of this function appears to be checking if something is enabled, and returning a boolean result. What do you think of renaming this to `enabled` or `isEnabled`?